src/main/java/com/mixpanel/android/viewcrawler/ViewVisitor.java

```diff
@@ -38,7 +38,7 @@
         public void OnEvent(View host, String eventName, boolean debounce);
```

Comment:
Can we make CycleDetector a static class in this PRQ? (It may have to move out of LayoutUpdateVisitor but it can still be private inside of the ViewVisitor class)